book/website/ethical-research/ethics-open-source-governance.md

```diff
@@ -0,0 +1,99 @@
+
+For many projects that are community led, this means involving the people that are affected by the project’s outcome in the design, development, and deployment stages.
+True co-creation is an active process of inclusion, and there are many levels of engagement that can be tailored to a projects’ individual needs (see Figure 1. Arnstein's Ladder of Citizen Participation), which a governance model might also need to take into account, for example, how is delegated power dealt with by the project?  
+
+
+```{figure} ../../figures/ladder-of-participation.jpg
+---
+name: ladder-of-participation
+alt: Colourful graphic of Arnstein's Ladder of Participation (1969) drawn by Juliet Young. A rainbow ladder describes the different types of participation in research - manipulation, educating, informing, consultation, placation, partnership, delegated power and citizen control. 
+---
+Arnstein's Ladder of Citizen Participation, illustration credit: [@clinical.creative.psychologist (Juliet Young)](https://twitter.com/juliet_young1/status/1384604477697761281)
+```
+
+No open source project is an island, and the goal of making projects open source is to explicitly invite the use and collaboration on a project which leads to a community of contributors and collaborators making the world a better place, one commit at a time.
+It's important that the governance model you select provides robust mechanisms for the level of community participation you are hoping to achieve with your project, with self-reflection being a useful starting point to guide your decision making. 
+
+(er-ethics-open-source-governance-continuous-process)=
+## Ethical Governance is a Continuous Process
+There are many ways in which your project can (and probably will) change over its lifetime, and some of these changes may be unexpected. 
+For example, your software might start being used in domains you have not considered, it might experience a faster growth rate than you have expected, or you may need to decide how to handle an influx of new contributors.
+
+This may also mean that you have to change the way that the project is run, to actively include new parts of the community, deal with a surge in contributions, or face new challenges.
+In this scenario, try and reflect the ways in which the governance of your project should evolve to reflect the ethics you commit to.
+For instance, is your governance structure built to allow the whole community to participate and influence a project, that will in turn influence themselves and their work?
+
+It may feel like a giant leap to go from, say, a free-form do-ocracy to something more structured, or to give up your position of benevolent dictator, or to introduce different voting rules for the steering committee, but if you want your project to reflect your ethics, you will need to allow your ethics to impact the project, and iterate on this process. 
```

Comment:
```suggestion
It may feel like a giant leap to go from, say, a free-form do-ocracy to something more structured, or to give up your position of benevolent dictator, or to introduce different voting rules for the steering committee.
However, if you want your project to reflect your ethics, you will need to allow your ethics to impact the project and iterate on this process. 
```